### PR TITLE
Add `GeoJSONGeometryCollection` interface to wellknown.

### PR DESCRIPTION
**Pull request by Maldron**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/mapbox/wellknown/blob/master/index.js#L191
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## wellknown ([<kbd>unpkg</kbd>](https://unpkg.com/browse/wellknown@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'wellknown'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0f9a4b2d90e3bdf9fd7483394f67495e417366f7
</p>

File: types/wellknown/wellknown-tests.ts [hunk 1]
```diff
@@ -8,3 +8,21 @@ const geoJson: wellknown.GeoJSONGeometry = {
 };
 
 wellknown.stringify(geoJson); // $ExpectType string
+
+wellknown.parse("GEOMETRYCOLLECTION (1 2)"); // $ExpectType GeoJSONGeometry
```
Comment (davodesign84) [inline types/wellknown/wellknown-tests.ts:12]: I haven't worked with WKT strings for a while but shouldn't this be something like `GEOMETRYCOLLECTION( POINT(1, 2))` ?

Comment (Maldron) [reply types/wellknown/wellknown-tests.ts]: ...yes, yes it should. 🤦  Thanks, fixed.

**Review by davide-scalzo — APPROVED** [review]
Looks alright to me

**Comment by Maldron** [comment]
Ready to merge

**Pull request merged**